tools/rosgraph/src/rosgraph/impl/graph.py

```diff
@@ -200,9 +203,9 @@ def __init__(self, start, end, label=''):
         self.start = start
         self.end = end
         self.label = label
-        self.key = "%s|%s"%(self.start, self.label)
+        self.key = (self.start, self.label)
```

Comment:
Darn, I can't really argue with the fact that people _might_ be relying on this. Is `|` a valid character in node or edge names?